### ISSUE TITLE
Generated TypeScript test file(s) do not run

### ISSUE THREAD

**Issue by iva2k** [feathers-plus/cli#8]
Generated TypeScript test file(s) do not run

Using Windows7, node v8.11.2, Performed the following steps from [https://generator.feathers-plus.com/get-started/](https://generator.feathers-plus.com/get-started/)

```bash
npm i -g @feathers-plus/cli
mkdir feathers-app
cd feathers-app
feathers-plus generate options
# chosen all defaults, except "Generate TypeScript code?" Yes

feathers-plus generate app
# chosen all defaults

npm start
# runs ok
```

Trying test:
```
npm test
# Executes 0 tests
```

**Comment by iva2k** [comment]
Details:
```
npm test > log.test 2>&1
dir test >> log.test
# Fails
```

Test does not seem to find the generated test file "test/app.test.ts", here's the log:

log.test
```

> api@0.0.0 test C:\dev\ionic3-feathersplus-try2\api.ts
> npm run tslint && npm run mocha


> api@0.0.0 tslint C:\dev\ionic3-feathersplus-try2\api.ts
> tslint -p tsconfig.json -c tslint.json && tslint -p tsconfig.test.json -c tslint.json


> api@0.0.0 mocha C:\dev\ionic3-feathersplus-try2\api.ts
> ts-mocha -p tsconfig.test.json 'test/**/*.test.ts' --timeout 10000 --exit

Warning: Could not find any test files matching pattern: 'test/**/*.test.ts'


  0 passing (1ms)

 Volume in drive C is OS
 Volume Serial Number is 64F2-D78A

 Directory of C:\dev\ionic3-feathersplus-try2\api.ts\test

07/17/2018  02:28 PM    <DIR>          .
07/17/2018  02:28 PM    <DIR>          ..
07/17/2018  02:26 PM             1,425 app.test.ts
               1 File(s)          1,425 bytes
               2 Dir(s)  229,470,457,856 bytes free
```

**Comment by iva2k** [comment]
After ```feathers-plus generate authentication```, one more test file is added (test/services/users.test.ts), but ```npm test``` still sees 0 files.

**Comment by j2L4e** [comment]
do they run when you regenerate in JS rather than TS?

**Comment by iva2k** [comment]
Yes, JS tests run fine.

**Comment by iva2k** [comment]
It looks like on Windows, using single quotes in commands is a no-no:

```
C:\~~\api>ts-mocha -p tsconfig.test.json 'test/**/*.test.ts' --timeout 10000 --exit
Warning: Could not find any test files matching pattern: 'test/**/*.test.ts'

  0 passing (1ms)
```

Compare to double quotes:
```
C:\~~\api>ts-mocha -p tsconfig.test.json "test/**/*.test.ts" --timeout 10000 --exit


  Feathers application tests
    √ starts and shows the index page (48ms)
    404
error: Page not found {"type":"FeathersError","name":"NotFound","code":404,"className":"not-found","data":{"url":"/path/to/nowhere"},"errors":{}}
      √ shows a 404 HTML page
error: Page not found {"type":"FeathersError","name":"NotFound","code":404,"className":"not-found","data":{"url":"/path/to/nowhere"},"errors":{}}
      √ shows a 404 JSON error without stack trace

  3 passing (151ms)
```

Runs ok.

In package.json changed this line in "scripts":
```
    "mocha": "ts-mocha -p tsconfig.test.json 'test/**/*.test.ts' --timeout 10000 --exit",
```

to (escaped double-quotes): 
```
    "mocha": "ts-mocha -p tsconfig.test.json \"test/**/*.test.ts\" --timeout 10000 --exit",
```
Got ```npm test``` running ok:
```
...
  3 passing (124ms)
```

**Comment by eddyystop** [comment]
This is awkward. We need to generate one script that runs on Linux, Windows, OS.

I believe the single quotes were needed to run the globs properly on Linux. @j2L4e would double quotes work on Linux?

**Comment by j2L4e** [comment]
Yep, works :+1:

**Comment by eddyystop** [comment]
Will be published in a few hours as 0.4.17.
